classes/tombstone/DataObjectTombstoneSettingsDAO.inc.php

```diff
@@ -34,7 +34,7 @@ function &getSetting($tombstoneId, $name, $locale = null) {
 		while (!$result->EOF) {
 			$row = $result->getRowAssoc(false);
 			$value = $this->convertFromDB($row['setting_value'], $row['setting_type']);
-			if ($row['locale'] == '') $setting[$name] = $value;
+			if (!array_key_exists('locale', $row) || $row['locale'] == '') $setting[$name] = $value;
```

Comment:
I can try if that works, but it complains that index 'locale' doesn't exist in the $row array. Thus maybe it could complain again, because it will try to get that array element by the index that doesn't exist, or?